jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_warning/diagnos_ether_listener.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import rospy
+from diagnostic_msgs.msg import DiagnosticArray
+
+hoge = None
+
+def callback(data):
+    global hoge
```

Comment:
what is this `hoge` for?

Reply:
I'm sorry, the variables I was using during programming remained. I will fix it!